apps/template/src/App.tsx

```diff
@@ -39,20 +39,17 @@ export const App = (props: AppProps) => {
   })
 
   const fallbackRender = ({ error }: { error: { message: string } }) => {
-    return (
-      <div className="w-1/2">
-        <CodeBlock className={preErrorClasses} copy={false}>
-          {error?.message || error?.toString() || "An error occurred"}
-        </CodeBlock>
-      </div>
-    )
+    return <div className="w-1/2">{error?.message || error?.toString() || "An error occurred"}</div>
   }
 
   return (
     <QueryClientProvider client={queryClient}>
       <AppShell
         pageHeader="Converged Cloud | App Template"
         embedded={props.embedded === "true" || props.embedded === true}
+        fullWidthContent={false}
```

Comment:
**(Change Required ?)**

When `embedded` is `true`, default `fullWidthContent` in `AppShell` is `true`. And when not, `false`. 
Is this change just to fix the TS issues? If so, since it's not part of requirements, let's just assign `undefined`, to ensure we're not changing any functionality? Or are we sure it should be `false`?

Reply:
Still same, can you try revert? 🙏

Reply:
Still same, can you try revert? 🙏

Reply:
Still there